jabgui/src/main/java/org/jabref/Launcher.java

```diff
@@ -87,7 +89,7 @@ public static void main(String[] args) {
             systemExit();
         }
 
-        PreferencesMigrations.runMigrations(preferences);
+        PreferencesMigrations.runMigrations(preferences, bibEntryTypesManager);
```

Comment:
Remove the parameter.

Later, use `preferences.getCustomEntryTypesRepository()` to get the `bibEntryTypesManager`